### DISCUSSION TITLE
How to get games to open on virtual display screen instead of screen 1?

### DISCUSSION THREAD

**Discussion by justifun** [ClassicOldSong/Apollo#131] [Q&A]
When i launch moonlight on my client, and select the virtual display, any app or game i launch still opens on screen 1 instead of the virtual display.  How do i get them to show up on the virtual display screen instead?

**Comment by ClassicOldSong** [comment]
Change the newly created virtual display as primary display in Window settings. The next time you stream it'll automatically be set to primary and your game will launch on the virtual display.

**Comment by justifun** [comment]
Will it fall back to my main screen 1 monitor on my gaming computer when it
disconnects?  I have 2 monitors on the main computer, so hopefully It won't
pick #2

**Reply by ClassicOldSong** [reply]
It will remember your selection with monitor combinations. Apollo let Windows handle all these, so you just need to refer to standard Windows behavior.
